### DISCUSSION TITLE
Persist Selected Keys

### DISCUSSION THREAD

**Discussion by michele-grifa** [rappasoft/laravel-livewire-tables#639] [Q&A]
Is possible to persist the selected keys? 
Actually to save selected state,  in my model i have a `selected` column, but i don't know how to bind the checkbox with my column.

**Comment by rappasoft** [comment]
It wasn't built to do that, but you can _try_ to override the `selectedKeys` method and return an array of the keys to be selected. Not sure if it would work though.

**Reply by michele-grifa** [reply]
I tried but the `selectedKeys` method never get called. I also tried to override `getSelected` method to show default selected values but not work neither
